filestack/utils.py

```diff
@@ -30,8 +32,10 @@ def handle_request(self, name, *args, **kwargs):
         requests_method = getattr(original_requests, name)
         response = requests_method(*args, **kwargs)
 
-        if not response.ok:
-            raise Exception(response.text)
+        try:
+            response.raise_for_status()
+        except HTTPError as e:
+            raise FilestackHTTPError(response.text) from e
```

Comment:
Added the `raise from` to keep exception chaining.


Ideally i would have liked `FilestackHTTPError` to inherit from `requests.exceptions.HTTPError`. Then i would need to weave response.text into the existing http error message. But that would change the test case asserts all over the place, and maybe users also are expecting the error messages to have a structure. This way we keep compatability.